spec/controllers/my/files_controller_spec.rb

```diff
@@ -9,6 +9,13 @@
     end
   end
 
+  let(:other_collection) do
+    Collection.new(title: 'other test collection').tap do |c|
```

Comment:
This doesn't seem to work. A collection is not valid until the depositor metadata has been applied. So the create alone will not save the collection and it'll still need a save in the tap.